modules/sdk-coin-ethlike/src/lib/transactionBuilder.ts

```diff
@@ -0,0 +1,29 @@
+import EthereumCommon from '@ethereumjs/common';
+import { TransactionBuilder as AbstractTransactionBuilder, Transaction } from '@bitgo/abstract-eth';
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { BuildTransactionError, TransactionType } from '@bitgo/sdk-core';
+import { TransferBuilder } from './transferBuilder';
+
+export class EthLikeTransactionBuilder extends AbstractTransactionBuilder {
+  protected _transfer: TransferBuilder;
+
+  constructor(_coinConfig: Readonly<CoinConfig>, common?: EthereumCommon) {
+    super(_coinConfig);
+    if (!common) {
+      throw new Error('Common must be provided for EthLikeTransactionBuilder');
+    }
```

Comment:
Can we not make the `common` parameter an optional argument since we are throwing an error if it is not passed? Or reverse

Reply:
'common` as an optional parameter and we cannot make it mandatory cause it will break other coins functionality